webapp/src/pages/tagConfig/components/TagConfigComponent.js

```diff
@@ -0,0 +1,375 @@
+import React, { Component } from "react";
+import { tagsService } from "../../../services/tags";
+import { usePagination, useTable, Column } from "react-table";
+import { Link } from "react-router-dom";
+import { withRouter } from "react-router";
+import { withTranslation } from 'react-i18next';
+import FormTextBox from "../../../components/form/FormTextBox";
+import FormTextArea from "../../../components/form/FormTextArea";
+import FormSelect from "../../../components/form/FormSelect";
+
+const TAG_TYPES = ["RESPONSE", "REGISTRATION"];
+
+class TagConfigComponent extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      tags: [],
+      updatedTag: {
+        name: {},
+        tag_type: "",
+        description: {},
+        active: true
+      },
+      isMultiLingual: this.props.organisation.languages.length > 1,
+      isValid: false,
+      loading: false,
+      error: "",
+      errors: [],
+      showErrors: false,
+      tagEntryVisible: false
+    };
+  }
+
+  componentDidMount() {
+    if (this.props.event) {
+      tagsService.getTagListConfig(this.props.event.id).then(result => {
+        this.setState({
+          loading: false,
+          tags: result.tags,
+          error: result.error
+        });
+        console.log(this.state.tags);
+      });
+    }
+  }
+
+  addNewTag = () => {
+    const errors = this.validateTagDetails();
+    if (errors.length === 0) {
+      tagsService.addTag(this.state.updatedTag, this.props.event.id).then(result => {
+        if (result.error) {
+          this.setState({
+            errors: [this.props.t(result.error)],
+            showErrors: true
+          });
+        }
+        else {
+          //this.props.history.goBack();
+        }});
+    }
+    else {
+      console.log('post failed');
+      this.setState({
+        showErrors: true,
+        errors: [this.props.t(errors)]
+      });
+    }
+  };
+
+  updateExistingTag = () => {
+    const errors = this.validateTagDetails();
+    if (errors.length === 0) {
+      tagsService.updateTag(this.state.updatedTag, this.props.event.id).then(result => {
+        if (result.error) {
+          this.setState({
+            errors: [this.props.t(result.error)],
+            showErrors: true
+          });
+        }
+        else {
+          //go back to tag table
```

Comment:
You'll need to update the tags state to see the change in the table :) 
see how its done in InvitedGuestComponent line 280 for an example